### PR TITLE
Add clang format action

### PR DESCRIPTION
**Pull request by Myoldmopar**
Pull request overview
---------------------
 - Adds Clang-Format builds to our Github Actions to verify every push/PR has valid style in the src/EnergyPlus and tst/EnergyPlus/unit directories.

### CODE REVIEW

File: .github/workflows/clang-format-check.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+        path:
+          - 'src/EnergyPlus'
+          - 'tst/EnergyPlus/unit'
+    steps:
+      - uses: actions/checkout@v2
+      - name: Run clang-format style check for C/C++ programs.
+        uses: jidicula/clang-format-action@v3.2.0
+        with:
+          clang-format-version: '10'
+          check-path: ${{ matrix.path }}
```
Comment (Myoldmopar) [inline .github/workflows/clang-format-check.yml:19]: New Github Action workflow.

File: scripts/dev/verify_file_formatting.sh [hunk 2]
```diff
@@ -1,21 +0,0 @@
-#!/bin/bash
```
Comment (Myoldmopar) [inline scripts/dev/verify_file_formatting.sh:1]: Removed these 2 files that we previously used to check formatting.

File: src/ConvertInputFormat/main.cpp [hunk 3]
```diff
@@ -70,14 +70,12 @@ enum class OutputTypes
     BSON
 };
 
-template <typename T>
-void displayMessage(T t)
+template <typename T> void displayMessage(T t)
```
Comment (Myoldmopar) [inline src/ConvertInputFormat/main.cpp:73]: Small non-functional code changes to bring the code up to our Clang-Format style.

**Review by Myoldmopar — COMMENTED** [review]
This should be ready to go in now.  Thanks for all the work getting it going on Github Actions @jmythms

**Comment by Myoldmopar** [comment]
Assuming CI is all green this should merge later this morning.

**Comment by Myoldmopar** [comment]
All good here, I need to tweak the integration coverage limit but this is good.  Thanks for fixing this stuff @jmythms

**Comment by jmythms** [comment]
Thank you very much!

**Pull request merged**